lib/issues.go

```diff
@@ -77,63 +91,33 @@ func DidIssueChange(config cfg.Config, ghIssue github.Issue, jIssue jira.Issue)
 
 	anyDifferent = anyDifferent || (ghIssue.GetTitle() != jIssue.Fields.Summary)
 	anyDifferent = anyDifferent || (ghIssue.GetBody() != jIssue.Fields.Description)
-
-	key := config.GetFieldKey(cfg.GitHubStatus)
-	field, err := jIssue.Fields.Unknowns.String(key)
-	if err != nil || *ghIssue.State != field {
-		anyDifferent = true
-	}
-
-	key = config.GetFieldKey(cfg.GitHubReporter)
-	field, err = jIssue.Fields.Unknowns.String(key)
-	if err != nil || *ghIssue.User.Login != field {
-		anyDifferent = true
-	}
-
-	labels := make([]string, len(ghIssue.Labels))
+	anyDifferent = anyDifferent || jiraCustomFieldsNeedUpdate(config, jIssue, cfg.GitHubStatus, ghIssue.GetState())
```

Comment:
A helper method here seems promising.

Reply:
I don't understand this comment. Are you saying that the `jiraCustomFieldNeedsUpdate`  helper method is good or is that you want to see another one for some other purpose?

Reply:
Again here, I'm reacting to seeing a long string of exactly duplicated code in 3 lines. In this case, a bunch of it is the boilerplate "anyDifferent = anyDifferent ||", which is unfortunate. My mind starts wondering if there is a way to declare the stuff you want to compare, and then run through a loop instead of a series of statements. But I'm not sure how practical that is. 

I'm inclined to leave this one, since I can't think of an obvious way to reduce the amount of duplication. But that doesn't mean I have to like it. :)